code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3168,56 +3155,9 @@ datum
 				if (!volume_passed)
 					return ..()
 				if (method == INGEST)
-					var/datum/ailment/addiction/AD = M.addicted_to_reagent(src)
-					if (!AD)
-						boutput(M, "<B>THIS TASTES <font size='92'>~<font color='#FF0000'> A<font color='#FF9900'> M<font color='#FFff00'> A<font color='#00FF00'> Z<font color='#0000FF'> I<font color='#FF00FF'> N<font color='#660066'> G<font color='#000000'> ~ !</font></B>")
+					boutput(M, "<B>THIS TASTES <font size='92'>~<font color='#FF0000'> A<font color='#FF9900'> M<font color='#FFff00'> A<font color='#00FF00'> Z<font color='#0000FF'> I<font color='#FF00FF'> N<font color='#660066'> G<font color='#000000'> ~ !</font></B>")
 				..(M, method, volume_passed)
 
-			on_mob_life(var/mob/M, var/mult = 1)
```

Comment:
this removes the point of the *secret chem*, high healing while also being addictive